### DISCUSSION TITLE
No active codespaces but cannot create new

### DISCUSSION THREAD

**Discussion by MOlausson** [community/community#26913] [Codespaces]
<p>I’ve been fiddling with codespaces today and may have tripped over something. I’ve deleted my codespaces but now I cannot create anything from the UI:</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/a/af9b7d55d222760d374528b2328ee54065bdb939.png" data-download-href="/uploads/short-url/p3uGUYgUMpQv7TPOa0tWGnNKxHb.png?dl=1" title="image" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/7904771/181100595-876cce56-e5cf-4a40-bd33-b24280d850b0.png" alt="image" data-base62-sha1="p3uGUYgUMpQv7TPOa0tWGnNKxHb" width="690" height="141" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/a/af9b7d55d222760d374528b2328ee54065bdb939_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image</span><span class="informations">841×173 8.36 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>Is there another way to see what can be blocking me here?</p>

**Comment by matthewisabel** [comment]
<p>We’re looking into this to see what could be happening. It looks like it may be a bug but I’ll update you as soon as we have it sorted.</p>

**Comment by MOlausson** [comment]
<p><a class="mention" href="/matthewisabel">@matthewisabel</a> any progress on this?</p>

**Comment by aeisenberg** [comment]
<p>I had the same problem a short while ago. I usually sign into GitHub using SSO. When I accessed the codespaces page, I wasn’t completely authenticated and so the codespaces for my org were hidden. When I fully authenticated using my SSO provider, the hidden codespaces started showing.</p>
<p>Maybe this is your problem too?</p>

**Comment by matthewisabel** [comment, marked as answer]
<p>This should be resolved. Please let us know if there are issues here you’re continuing to see!</p>

**Comment by MOlausson** [comment]
<p>Works great now, thanks <a class="mention" href="/matthewisabel">@matthewisabel</a>!</p>

**Comment by sarisia** [comment]
<p>Hi,<br>
I’m encountering the same issue for about 2 weeks<br>
and it’s continuing.</p>

**Comment by matthewisabel** [comment]
<p>Ah, I’m sorry to hear that <a class="mention" href="/sarisia">@sarisia</a>. Let me look in to this to see what could still be happening.</p>

**Comment by matthewisabel** [comment]
<p>We had an additional fix roll out today if you don’t mind verifying that you can now create Codespaces <a class="mention" href="/sarisia">@sarisia</a></p>

**Comment by sarisia** [comment]
<p>Problem solved. Thank you for quick response!</p>
